src/Cache/MemoryCacheItemPool.php

```diff
@@ -55,12 +55,12 @@ public function getItem($key)
      *   key is not found. However, if no keys are specified then an empty
      *   traversable MUST be returned instead.
      */
-    public function getItems(array $keys = [])
+    public function getItems(array $keys = []): iterable
```

Comment:
Does the PHP Doc `@return` need to be updated? I'm not sure if there is a difference between `array` and `iterable` here.